### PR TITLE
[MTB] update kyverno policy - disallow root user

### LINKED ISSUE #1107: [MTB] Add Kyverno Policy - Require run as non-root user

### PR DESCRIPTION
**Pull request by realshuting**
Fixes #1107.

/assign @JimBugwadia

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ The following tests are currently defined (tests marked `pending` are planned fo
...
 * [Block Multitenant Resources](kubectl-mtb/test/benchmarks/block_multitenant_resources)
 * [Block add capabilities](kubectl-mtb/test/benchmarks/block_add_capabilities)
-* Require run as non-root user (**pending**)
-* [Require image pull `always`](kubectl-mtb/test/benchmarks/require_always_pull_image) (**pending**)
-* Require PVC reclaim policy `delete` (**pending**)
-* Require CAP_DROP_ALL (**pending**)
+* [Require image pull `always`](kubectl-mtb/test/benchmarks/require_always_pull_image)
+* [Require run as non-root user](kubectl-mtb/test/benchmarks/require_run_as_non_root_user)
+* Require PVC reclaim policy `delete`
+* Require CAP_DROP_ALL
```
Comment (rjbez17) [inline benchmarks/README.md:35]: This moved and the PVC reclaim moved from pending to not, but I don't see anything in the PR to test for it. I assume they were added in a different PR?

Comment (realshuting) [reply benchmarks/README.md]: Thanks for pointing it out!
It must be removed by accident, added them back.

**Review by JimBugwadia — APPROVED** [review]

**Pull request merged**